apps/frontend/src/libs/components/checkbox/checkbox.tsx

```diff
@@ -7,6 +7,7 @@ import {
 	type InputOption,
 } from "~/libs/types/types.js";
 
+import { Icon } from "../components.js";
```

Comment:
```suggestion
import { Icon } from "../icon/icon.js";
```
Avoid circular dependencies